packages/components/src/Breakout/Breakout.tsx

```diff
@@ -1,46 +1,10 @@
 import { ReactNode } from 'react';
-import { alignment, cn, createVar } from '@marigold/system';
 import type { AriaRegionProps } from '@marigold/types';
 
 export interface BreakoutProps extends AriaRegionProps {
   children?: ReactNode;
-
-  /**
-   * Set the height of the breakout element.
-   */
-  height?: string;
-
-  /**
-   * Horizontal alignment of the items inside the breakout element.
-   * @default left
-   */
-  alignX?: keyof typeof alignment.horizontal.alignmentX;
```

Comment:
why is this useless? With this props you can position the Breakout elements or not?

Reply:
Because you can just use another layout component to do that. Do you have an actual use case where this is useful?

Reply:
Actually I don't know who ever used our breakout :D